components/frontdoor/main.tf

```diff
@@ -0,0 +1,24 @@
+module "landing_zone" {
+  source = "git::https://github.com/hmcts/terraform-module-frontdoor.git?ref=master"
+
+  providers = {
+    azurerm      = azurerm
+    azurerm.data = azurerm.data
```

Comment:
I can't find this actually being used in the module, I don't think it's needed anymore? and should be removed from the module?

Reply:
terraform frontdoor module needs updating.